fortuna/.gitignore

```diff
@@ -1 +1,2 @@
 /target
+config.yaml
```

Comment:
i think you may also need to `git rm config.yaml`. I think it's probably still in the repo.